[ INFO ]- 开始从MetaNet数据库加载NgramDict
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 68, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp - 2019-04-01 07:46:15,582 - nvwa - logger.py:442
[ INFO ]- 从MetaNet数据库加载NgramDict完毕，没有可用的MetaNet
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 74, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp total_seconds:0.015 - 2019-04-01 07:46:15,596 - nvwa - logger.py:442
[ INFO ]- 开始加载数据库元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 159, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp - 2019-04-01 07:48:35,184 - nvwa - logger.py:442
[ INFO ]- 加载数据库元数据完毕，没有可用的元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 163, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp total_seconds:0.0 - 2019-04-01 07:48:35,184 - nvwa - logger.py:442
[ INFO ]- 开始从MetaNet数据库加载NgramDict
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 68, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp - 2019-04-01 07:48:35,200 - nvwa - logger.py:442
[ INFO ]- 从MetaNet数据库加载NgramDict完毕，没有可用的MetaNet
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 74, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp total_seconds:0.0 - 2019-04-01 07:48:35,200 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-04-01 07:48:57,404 - nvwa - logger.py:442
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 442, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-04-01 07:48:59,374 - nvwa - logger.py:442
[ INFO ]- 开始加载数据库元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 159, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp - 2019-05-30 20:21:26,313 - nvwa - logger.py:449
[ INFO ]- 加载数据库元数据完毕，没有可用的元数据
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 163, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 758, in loadAllMetaFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 746, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp total_seconds:0.026 - 2019-05-30 20:21:26,339 - nvwa - logger.py:449
[ INFO ]- 开始从MetaNet数据库加载NgramDict
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 68, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp - 2019-05-30 20:21:26,341 - nvwa - logger.py:449
[ INFO ]- 从MetaNet数据库加载NgramDict完毕，没有可用的MetaNet
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\metaEngine.py", line 74, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 783, in loadNgramDictFromDB
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\memory.py", line 747, in init
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\brain.py", line 63, in init
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 23, in setUp total_seconds:0.007 - 2019-05-30 20:21:26,348 - nvwa - logger.py:449
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-05-30 20:24:02,263 - nvwa - logger.py:449
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-05-30 20:24:11,207 - nvwa - logger.py:449
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-05-30 20:38:43,184 - nvwa - logger.py:449
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-05-30 20:38:50,013 - nvwa - logger.py:449
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-05-30 20:38:55,936 - nvwa - logger.py:449
[ INFO ]- 更改后受影响的条数(1)。
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 1003, in updateAllInDB
File "E:\nvwa-run\trunk\loongtian\nvwa\models\baseEntity.py", line 959, in updateAttributeValues
File "E:\nvwa-run\trunk\loongtian\nvwa\models\realObject.py", line 522, in setType
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 329, in _createNewPatternAndMeaning
File "E:\nvwa-run\trunk\loongtian\nvwa\engines\modelingEngine.py", line 230, in _createRealObjectExecutionInfo - 2019-05-30 20:39:11,569 - nvwa - logger.py:449
[ INFO ]- “组件属性父对象成分”思考出现错误，原因：0
InnerTraceback (most recent call last):
File "E:\nvwa-run\trunk\loongtian\util\log\logger.py", line 449, in info
File "E:\nvwa-run\trunk\loongtian\nvwa\organs\mind.py", line 109, in execute
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\thinkingCentral.py", line 63, in thinkStringInput
File "E:\nvwa-run\trunk\loongtian\nvwa\centrals\perceptionCentral.py", line 47, in receive
File "E:\nvwa-run\trunk\test\nvwa\testBrain.py", line 380, in testMeaning - 2019-05-30 20:39:16,328 - nvwa - logger.py:449
